src/DateRangePicker.jsx

```diff
@@ -156,7 +156,7 @@ export default class DateRangePicker extends PureComponent {
           returnValue="start"
           value={valueFrom}
         />
-        –
+        <span>–</span>
```

Comment:
I like this! Can you add a class name so it could be even easier?
```suggestion
        <span className={`${baseClassName}__divider`}>
          –
        </span>
```

Reply:
Nice. Done and done!